subsys/net/ip/dhcpv4.c

```diff
@@ -991,6 +992,17 @@ static enum net_verdict net_dhcpv4_input(struct net_conn *conn,
 static void dhcpv4_iface_event_handler(struct net_mgmt_event_callback *cb,
 				       u32_t mgmt_event, struct net_if *iface)
 {
+	sys_snode_t *node = NULL;
+
+	SYS_SLIST_FOR_EACH_NODE(&dhcpv4_ifaces, node) {
+		if (node == &iface->config.dhcpv4.node) {
+		  break;
```

Comment:
and your indentation is wrong: tabs only (plus spaces when it comes to parameters alignment) please.